packages/expect/src/__tests__/spyMatchers.test.ts

```diff
@@ -1013,21 +1251,31 @@ describe.each([
       // ignore error
     }
 
-    // It doesn't matter what return value is tested if the call threw
-    caller(jestExpect(fn).not[returnedWith], 'foo');
-    caller(jestExpect(fn).not[returnedWith], null);
-    caller(jestExpect(fn).not[returnedWith], undefined);
+    if (isNth(returnedWith)) {
+      // It doesn't matter what return value is tested if the call threw
+      jestExpect(fn).not[returnedWith](1, 'foo');
+      jestExpect(fn).not[returnedWith](1, null);
+      jestExpect(fn).not[returnedWith](1, undefined);
 
-    expect(() =>
-      caller(jestExpect(fn)[returnedWith], undefined),
-    ).toThrowErrorMatchingSnapshot();
+      expect(() =>
+        jestExpect(fn)[returnedWith](1, undefined),
+      ).toThrowErrorMatchingSnapshot();
+    } else {
+      // It doesn't matter what return value is tested if the call threw
+      jestExpect(fn).not[returnedWith]('foo');
+      jestExpect(fn).not[returnedWith](null);
+      jestExpect(fn).not[returnedWith](undefined);
+
+      expect(() =>
+        jestExpect(fn)[returnedWith](undefined),
+      ).toThrowErrorMatchingSnapshot();
+    }
   });
 
-  const basicReturnedWith = ['toHaveReturnedWith', 'toReturnWith'];
-  if (basicReturnedWith.indexOf(returnedWith) >= 0) {
+  if (!isNth(returnedWith)) {
```

Comment:
`'lastReturnedWith'` and `'toHaveLastReturnedWith'` are currently not included. New snaps got generated for these cases.

Reply:
can we land that in a separate PR? Just to keep the noise down in this one